python/lsst/afw/cameraGeom/testUtils.py

```diff
@@ -419,20 +419,29 @@ def makeTestRepositoryItems(self, isLsstLike=False):
 
 @inTestCase
 def compare2DFunctions(self, func1, func2, minVal=-10, maxVal=None, nVal=5):
```

Comment:
I considered leaving this function as it was and adding a new `compare2DFunctionsVector` but in lsst_distrib everything that calls this function is calling it with funcs that are vectorized so everything worked anyhow. @ktlim if you are worried about compatibility I can leave this unchanged and add the new method.

Reply:
If it passes all the tests I'm okay with it.